ludwig/backend/horovod.py

```diff
@@ -37,26 +37,28 @@ class HorovodBackend(LocalPreprocessingMixin, Backend):
 
     def __init__(self, **kwargs):
         super().__init__(dataset_manager=PandasDatasetManager(self), **kwargs)
-        self._horovod = None
+        self._distributed = None
```

Comment:
Should `self._distributed` be an attribute of the `Backend` class?

Reply:
Only for Horovod. In the case of Ray, we are not in the right context from the backend to have access to the distributed api.